### PR TITLE
Add ReflectionFilter

### PR DESCRIPTION
**Pull request by finscn**
preview : 
image

online demo : http://fatidol.com/pixi-filters

### CODE REVIEW

File: filters/reflection/LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+The MIT License
+
+Copyright (c) 2013-2017 Mathew Groves, Chad Engler
```
Comment (cursedcoder) [inline filters/reflection/LICENSE]: I think you can replace this with your name.

Comment (finscn) [reply filters/reflection/LICENSE]: Thanks. @bigtimebuddy @englercj @GoodBoyDigital  , Shall I ?

Comment (bigtimebuddy) [reply filters/reflection/LICENSE]: Yes that’s fine with me

Comment (finscn) [reply filters/reflection/LICENSE]: @bigtimebuddy  , OK , I'll create a new PR for changing all filters created be me.

**Pull request merged**